src/geff/networkx/io.py

```diff
@@ -207,8 +212,9 @@ def _set_attribute_values(
             # Get either individual item or list instead of setting with np.array
             val = val.tolist() if val.size > 1 else val.item()
             if nodes:
-                if name == "position" and "position_attr" in graph.graph:
-                    graph_attr = graph.graph["position_attr"]
+                # Do subsequent todos needs to be changed with optional position?
+                if name == "position" and "position_attr" in graph.graph:  # TODO
+                    graph_attr = graph.graph["position_attr"]  # TODO
```

Comment:
@cmalinmayor Does this code block also need to be changed in order to make position optional?

Reply:
It should be removed, because it is no longer needed. This is the part I added to rename the stored position (always called "position") back to the original name (the `position_attribute`) when we read the graph back in. Since we now save it with the `position_attribute` name, we don't need to handle it differently anymore.